src/py-opentimelineio/opentimelineio/adapters/svg.py

```diff
@@ -1094,6 +1083,49 @@ def _draw_collection(collection, svg_writer, extra_data=()):
     pass
 
 
+def _available_range_from_clip(clip):
+    """
+    Default to a Clip's source range but
+    if the media reference has a defined available_range use that it instead
+
+    :param opentimelineio.schema.Clip clip:
+    :return: The clips available range
+    :rtype: opentimelineio.opentime.TimeRange
+    """
+    available_range = clip.source_range
+    if clip.media_reference.available_range:
+        available_range = clip.available_range()
+    return available_range
```

Comment:
This would read more clearly as something like:
```
if clip.media_reference and clip.media_reference.available_range:
    return clip.media_reference.available_range
else:
    return clip.source_range
```

Reply:
I landed on this because Pylint said the else was unnecessary.

```suggestion
    if clip.media_reference and clip.media_reference.available_range:
        return clip.available_range()
    return clip.source_range
```